docs/react-testing-library/api.mdx

```diff
@@ -370,6 +370,22 @@ test('returns logged in user', () => {
 })
 ```
 
+> NOTE: When using `renderHook` in conjunction with the `wrapper` and `initialProps` options, the `initialProps` are not passed to the `wrapper` component. To provide dependencies to the `wrapper` component, consider a solution like this:
+>
+> ```js
+> const createWrapper = (Wrapper, props) => {
+>   return function CreatedWrapper({ children }) {
+>     return <Wrapper {...props}>{children}</Wrapper>;
+>   };
+> };
+> 
+> ...
+> 
+> {
+>   wrapper: createWrapper(Wrapper, { dependency }),
```

Comment:
```suggestion
>   wrapper: createWrapper(Wrapper, { value: 'foo' }),
```

Clearer example. It wasn't clear where `dependency` was coming from. This example makes it look like we're injecting a context value